ivy/stateful/layers.py

```diff
@@ -8,6 +8,26 @@
 
 # ToDo: update docstrings and typehints according to ivy\layers
 
+# Identity #
+# -------#
+class Identity(Module):
```

Comment:
Actually we already have the [`Identity`](https://github.com/unifyai/ivy/blob/8ffd9ffad6f715a3b0bdedb3bbafe52d0085d2e6/ivy/stateful/layers.py#L2199) layer in this file, so we can use that instead